scripts/extensions/pool_update.precheck

```diff
@@ -100,17 +102,10 @@ def execute_precheck(session, control_package, yum_conf_file):
         else:
             raise PrecheckFailure()
     else:
-        lines = output.splitlines()
-        resultlines = [l for l in lines if l.startswith('<livepatch ')]
-        if resultlines:
-            result = resultlines[-1].split('\"')[1]
-            try:
-                return livepatch_dic[result]
-            except KeyError:
-                raise PrecheckFailure(
-                    'Install %s, precheck error: %s' % (control_package, result))
-        else:
-            return 'ok'
+        for key in livepatch_dic.keys():
+            if key in output:
+                return livepatch_dic[key]
+        return 'ok'
```

Comment:
This mechanism is common for both livepatch and non-livepatch packages. From xapi pool_update view, we don't know if it's livepatch or not. The package itself defines it.
Here the string parse is to ensure when precheck passes (yum install return value 0), the livepatch return string can be fetched and passed to the client that calls pool_update.precheck. If it's not livepatch, we return `ok`.
Based on design, if precheck passes, it should return enum value (one of the three) to xapi. They are what xapi can handle, an extra return value will fail xapi.

Does it make sense?
Thanks.